corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_forms.scss

```diff
@@ -52,6 +53,10 @@ legend {
   padding: $spacer 0;
   margin: 0 0 $spacer 0;
 
+  .field-control:first-child {
+    padding-left: ($grid-gutter-width - $spacer) / 2;
```

Comment:
Can you explain why use `($grid-gutter-width - $spacer) / 2` ? ( Just googled `$grid-gutter-width` and `$spacer`, but I'm still a bit confused. )

Reply:
those are variables from bootstrap 5, they store a value with units (`em`, I believe)

the form actions div adds `$spacer` sized padding to both sides of the row:
![Screenshot 2025-02-20 at 10 05 44 PM](https://github.com/user-attachments/assets/fcaf848b-8f4d-450c-81d9-77a3b069f23c)

this is what a normal row would look like:
![Screenshot 2025-02-20 at 10 00 06 PM](https://github.com/user-attachments/assets/81f65737-a1ba-47c2-b955-76414e4013a3)

each column has `$grid-gutter-width` of space in between